free-1.md

```diff
@@ -4,65 +4,176 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This project is designed to serve individuals who need to schedule meetings
+among a large group.
+
+Currently the de-facto standards for scheduling are one
+of the following two methods:
+ * Everyone send out their availability for a certain time and a leader
+picks the times that seem to work best for a plurality.
+ * Everyone uses some software, such as a WhenIsGood&#8482; or When2Meet&#8482;
+and choses a binary yes/no for each time slot.
+
+If I was able to help them, users would be able to specify, in a way that feels
+natural, their availability and preferences.  Hopefully this can reduce either
+the work of an individual (who must compile the results of an email) or the
+repetition inherent in painting over the same boxes for pairs of weeks when
+using a WhenIsGood&#8482;.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Since a large part of scheduling involves individuals
+expressing their constraints (and hopefully preferences as well), a language
+might be a good fit, since it can be tailored to allow this expressiveness
+in a way that a simple calendar might not.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I had the idea when (during the beginning of this school year) the many
+organizations of which I am a member began sending out scheduling notices,
+all trying to claim spots in my schedule simultaneously.  It irked me that
+there was not a better way to handle the scheduling problems, so I started
+thinking about how it might be improved.
+
+I am excited about trying to make the language integrate with existing calendar
+formats, since this would involve finding a clever abstraction for all the
+information which could serve as a medium between the natural / intuitive
+language with which users would describe their schedule, and the highly
+structured data of an exported Google Calendar or iCal instance.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Scheduling with precision and integration.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user might provide text of the form:
+```
+Thursday:
+  free one_hour_of(2pm - 5pm)
+  free 9pm - 11pm
+Friday:
+  busy 12pm - 12:30pm
+  busy after(7pm)
```

Comment:
Off the top of my head, I assume you would want some global defaults, and maybe consider things like holidays, time zones, and locales.